src/index.js

```diff
@@ -102,6 +120,11 @@ VideoStreamMerger.prototype.removeStream = function (mediaStream) {
 VideoStreamMerger.prototype.start = function () {
   var self = this
 
+  if (!supported) {
+    self.result = self._fallbackStream
```

Comment:
This won't be needed if we are just throwing an error. If Edge support is added, it should have the same functionality as other browsers.